.github/workflows/check-documentation.yml

```diff
@@ -0,0 +1,78 @@
+name: ocelot.social check documentation
+
+on:
+  push:
+    branches:
+      - 6816-refactor-docsocelotsocial-ensure-vuepress-does-not-break
```

Comment:
```suggestion
on: push
```

Reply:
Of course, this will be the last commit for this PR.